microsoft-edge/devtools-guide-chromium/storage/cookies.md

```diff
@@ -58,43 +57,46 @@ The **Cookies** table contains the following fields:
 
 *  **Size**.  The size, in bytes, of the cookie.
 
-*  **HTTP**.  If true, this field indicates that the cookie should only be used over HTTP, and JavaScript modification isn't allowed.  See [HttpOnly cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
+*  **HttpOnly**.  If true, this field indicates that the cookie should only be used over HTTP, and JavaScript modification isn't allowed.  See [HttpOnly cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
 
 *  **Secure**.  If true, this field indicates that the cookie must be sent to the server only over a secure, HTTPS connection.  See [Secure cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
 
 *  **SameSite**.  Contains `strict` or `lax` if the cookie is using the experimental [Samesite](https://developer.mozilla.org/docs/Web/HTTP/Cookies#SameSite_cookies) attribute.
 
-*  **Priority**.  Contains `low`, `medium` (default), or `high` if the cookie is using the deprecated [cookie Priority](https://bugs.chromium.org/p/chromium/issues/detail?id=232693) attribute.
+*  **SameParty**. This attribute provides web developers a means to annotate cookies that are allowed to be set or sent in same-party, cross-site contexts. 
+
+*  **Partition Key**. Partition key is the property or path within documents that can be used to distribute data. Data having the same partition key value are logically grouped together and stored in the same physical partition.
```

Comment:
```suggestion
*  **Partition Key**. A _partition key_ is the property or path within a document that can be used to distribute data within a partition. Multiple pieces of data that are spread throughout a document and that have the same partition key value are logically grouped together and stored in the same physical partition.
```
pls tech-review "within a partition".  Clarify "to distribute data"; what's the specific context/objective?  @captainbrosset

Reply:
I'm not an expert on this at all. So far, the simplest explanation I've read about this is from this section: https://developer.chrome.com/blog/chips-origin-trial/#what-is-chips

I think the partition key field in DevTools is meant to contain URLs (or at least domains, although I've not been able to test it successfully).
Partitioned cookies allow third-party websites to set cookies on client machines, but in a way that prevents other domains to read them.
The "Partition key" column in DevTools is way for users to see when a cookie is partitioned, and if so, which domain it belongs to.

Based on my understanding I think this part of the docs needs to be changed. But, again, I'm new to this, so can't help much more.

Reply:
@mikehoffms, @captainbrosset 
This is an area that I know nothing about. My initial definition is a copy from https://docs.microsoft.com/en-us/azure/cosmos-db/partitioning-overview.